v1/model.md

```diff
@@ -223,6 +224,7 @@ An Entry is identified by the following attributes:
 | documents            |      | array of resources | Related Documents                                                   |
 | externalIds          |      | array of resources | Related ExternalIds                                                 |
 | parties              |      | array of resources | Related Parties                                                     |
+| comments             |      | array of resources | Related Comments                                                    |
```

Comment:
You got any pointers for me as to where in the docs I should look to see how this would look in practise? Is it a plain array, or a more complex object telling how many comments there are etc?

Reply:
An example for each and every operation can be found in the [Postman collection](https://github.com/documaster/documaster-api/blob/develop/v1/postman/Documaster%20Integration%20API.postman_collection.json). You'll be able to find examples for fetching, creating, updating, and deleting stand-alone comments and also examples of creating a comment while creating an entry.

Alternatively, if you don't want to download the collection, you can take a look at the [advanced entry creation example](https://github.com/documaster/documaster-api/blob/develop/v1/examples.md#create-advanced) that creates parties. Comments are handled in exactly the same way as parties.

I'm open to suggestions about making this more visible and clear ;)

Reply:
Thanks!

I'm not too familiar with postman collections yet, but that advanced entry creation reference gives me a hint 👍 

Let me double check if I understand that reference correctly: `.data.comments` field would be an array of actual comments and `.data.resources.comments` would include additional meta data that comes in handy for pagination?

Reply:
Exactly!

Reply:
Great stuff, looks smashing 👍